torch/fx/passes/graph_drawer.py

```diff
@@ -140,12 +140,15 @@ def _get_leaf_node(
 
         def _typename(self, target: Any) -> str:
             if isinstance(target, torch.nn.Module):
-                return torch.typename(target)
-
-            if isinstance(target, str):
-                return target
+                ret = torch.typename(target)
+            elif isinstance(target, str):
+                ret = target
+            else:
+                ret = _get_qualified_name(target)
 
-            return _get_qualified_name(target)
+            # Escape "{" and "}" to prevent dot files like P522499127
```

Comment:
This paste is not accessible to OSS users. Please move it to e.g. a github gist

Reply:
Updated it to github gist!